content/pages/kontaktoss.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Kontakt Oss
+description: Kontaktinformasjon til Queer Hangout
+slug: /kontaktoss
+menu: 4
+---
+
+##Kontakt oss  
+
+Queer Hangout kan kontaktes på e-mail: queerhangout@posteo.no  
```

Comment:
Du kan skrive e-post som lenke ved å skrive `[queerhangout@posteo.no](mailto:queerhangout@posteo.no)`. En slik lenke vil åpne e-post klienten med e-post allerede fylt inn, som gjør det enklere for folk.

Vær obs på at vi kommer til å få mer spam når man legger ut e-post adresse